content/docs/faq.md

```diff
@@ -41,6 +41,112 @@ identity, and authorization features of modern Kubernetes. Helm’s permissions
 Cluster administrators can restrict user permissions at whatever granularity they see fit. Releases are still recorded
 in-cluster, and the rest of Helm’s functionality remains.
 
+### Improved Upgrade Strategy: 3-way Strategic Merge Patches
+
+Helm 2 used a two-way strategic merge patch. During an upgrade, it compared the most recent chart's manifest
+against the proposed chart's manifest (the one supplied during `helm upgrade`). It compared the differences between
+these two charts to determine what changes needed to be applied to the resources in Kubernetes. If changes were
+applied to the cluster out-of-band (like during a `kubectl edit`), those changes were not considered. This resulted in
+resources being unable to roll back to its previous state: because Helm only considered the last applied chart's
+manifest as its current state, if there were no changes in the chart's state, the live state was left unchanged.
+
+In Helm 3, Helm now uses a three-way strategic merge patch. Helm now considers the old manifest, its live state, and the
+new manifest when generating a patch.
+
+Let's go through a few common examples what this change impacts.
+
+#### Example: Rolling back where live state has changed
```

Comment:
Formatting nit: Should `Examples` be an H4 and all of the example titles H5?

Reply:
Good call.